### DISCUSSION TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'localsInner')

### DISCUSSION THREAD

**Discussion by kobe651jp** [ueberdosis/tiptap#3624] [General Questions]
This error occurs when processing a Table/TableCell. Has anyone encountered this error before? A search yields no hits.

![image](https://user-images.githubusercontent.com/44263972/212209974-108e31f0-6609-4e60-84ce-32247fb5d078.png)

![image](https://user-images.githubusercontent.com/44263972/212210059-f58ff121-f77c-4ac2-bd31-09b1a00c6070.png)

**Comment by linuz90** [comment]
I am facing the same issue as soon as I plug in the collaboration cursor (the collaboration works fine):

```
CollaborationCursor.configure({
  provider: hocuspocusProvider,
  user: {
    name: user?.displayName ?? "Loading...",
    color: colorRef.current,
  },
})
```

**Comment by vpontis** [comment]
@kobe651jp and @linuz90  can you share the versions of Prosemirror you have installed? I think this may be an issue with `prosemirror-view@1.30.0`  but I'm not sure why it would break this...

**Comment by kobe651jp** [comment]
prosemirror-view@1.29.1

**Comment by philipaarseth** [comment]
overriding prosemirror-view to 1.30.1 worked for me

**Reply by svenadlung** [reply]
Just for the sake of completeness:

> This most likely has to do with the use of different ProseMirror package versions.
> 
> Please do the following:
> 
> - always use @tiptap/pm if you want to use ProseMirror functions
> - update tiptap packages to version *2.2.1*, update tiptap-pro packages to version *2.6.4*
> - clean up the lock file if necessary
> 
> If there are still problems, please check whether different versions of ProseMirror packages are registered somewhere, e.g. with `npm ls prosemirror-view`.

**Reply by svenadlung** [reply]
Sorry, wasn't meant as a direct reply to Philip's message, more in general 😉

**Comment by AvaniBaheti** [comment]
Also check in which folder you are having required dependencies, first I was having it in outer folder's package.json and after I installed them in inner folder where I am having my Collab configuration it worked.

**Comment by markdon** [comment]
Had this error when moving a mouse over a table in tiptap and it's now fixed, thanks!
<details>
  <summary>Before `npm ls prosemirror-view`</summary>
<pre>
└─┬ @tiptap/pm@2.11.0
  ├─┬ prosemirror-dropcursor@1.8.1
  │ └── prosemirror-view@1.34.3
  ├─┬ prosemirror-gapcursor@1.3.2
  │ └── prosemirror-view@1.34.3
  ├─┬ prosemirror-history@1.4.1
  │ └── prosemirror-view@1.34.3
  ├─┬ prosemirror-state@1.4.3
  │ └── prosemirror-view@1.34.3
  ├─┬ prosemirror-tables@1.6.1
  │ └── prosemirror-view@1.34.3
  ├─┬ prosemirror-trailing-node@3.0.0
  │ └── prosemirror-view@1.37.1 deduped
  └── prosemirror-view@1.37.1 
</pre>
</details>


added to package.json
```
  "resolutions": {
    "prosemirror-view": "1.37.0"
  }
```
`yarn`
<details>
  <summary>After `npm ls prosemirror-view`</summary>
<pre>
└─┬ @tiptap/pm@2.11.0
  ├─┬ prosemirror-dropcursor@1.8.1
  │ └── prosemirror-view@1.37.0 deduped
  ├─┬ prosemirror-gapcursor@1.3.2
  │ └── prosemirror-view@1.37.0 deduped
  ├─┬ prosemirror-history@1.4.1
  │ └── prosemirror-view@1.37.0 deduped
  ├─┬ prosemirror-state@1.4.3
  │ └── prosemirror-view@1.37.0 deduped
  ├─┬ prosemirror-tables@1.6.1
  │ └── prosemirror-view@1.37.0 deduped
  ├─┬ prosemirror-trailing-node@3.0.0
  │ └── prosemirror-view@1.37.0 deduped
  └── prosemirror-view@1.37.0
</pre>
</details>

And the error is gone.
